src/api/controllers/UserReviewController.ts

```diff
@@ -0,0 +1,38 @@
+import { Body, CurrentUser, Delete, Get, JsonController, Params, Post } from 'routing-controllers';
+import { 
+    CreateUserReviewRequest,
+    GetUserReviewResponse,
+    GetUserReviewsResponse
+} from 'src/types';
+import { UserModel } from 'src/models/UserModel';
+import { UserReviewService } from '../../services/UserReviewService'
+import { UuidParam } from '../validators/GenericRequests';
+
+@JsonController('userReview/')
+export class UserReviewController {
+    private userReviewService: UserReviewService;
+
+    constructor(userReviewService: UserReviewService) {
+        this.userReviewService = userReviewService;
+    }
+
+    @Get()
+    async getUserReviews(): Promise<GetUserReviewsResponse> {
+        return { userReviews: await this.userReviewService.getAllUserReviews() };
+    }
+
+    @Get('id/:id/')
+    async getUserReviewsById(@Params() params: UuidParam): Promise<GetUserReviewResponse> {
+        return { userReview: await this.userReviewService.getUserReviewById(params)}
```

Comment:
space before the ending curly brace and semicolon at the end of this line - let's keep styling consistent